h/services/auth_ticket.py

```diff
@@ -16,19 +16,21 @@ class AuthTicketService:
     def __init__(self, session, user_service):
         self._session = session
         self._user_service = user_service
-        self._user = None
+        self._ticket = None
 
-    def verify_ticket(self, userid: str | None, ticket_id: str | None) -> User | None:
+    def verify_ticket(
+        self, userid: str | None, ticket_id: str | None
+    ) -> AuthTicket | None:
```

Comment:
`AuthTicketService.verify_ticket()` had to be changed to return the `AuthTicket` rather than the `User`, because the caller code (see above) now needs the auth ticket. The user is still available to the caller via `AuthTicket.user`.

Reply:
The code does do a DB lookup that won't be cached, so this might be a reasonable thing to do if there is a chance that a request might access the same ticket multiple times. I'm not familiar enough with the code to know if that is the case.